src/ru/skypro/Main.java

```diff
@@ -1,8 +1,84 @@
 package ru.skypro;
 
+import java.util.Arrays;
+
 public class Main {
+    private static final Employee[] employees = new Employee[10];
 
+    public static void FullName(Employee[] e) {
+        String fullname;
+        for (Employee fu : employees) {
+            fullname = fu.fullName;
+            System.out.println("Ф.И.О. - " + fullname);
+        }
+    }
+    public static void Sum (Employee[] e) {
+        double sum;
+        double summ = 0;
+        for (Employee su : employees) {
+            sum = su.salary;
+            summ += sum;
```

Comment:
избыточная вторая переменная
```suggestion
            summ = su.salary;
```